addons/mail/models/mail_thread.py

```diff
@@ -1985,7 +1985,8 @@ def _notify_thread(self, message, msg_vals=False, **kwargs):
             message_values['channel_ids'] = [(6, 0, [r['id'] for r in rdata['channels']])]
 
         self._notify_record_by_inbox(message, rdata, msg_vals=msg_vals, **kwargs)
-        self._notify_record_by_email(message, rdata, msg_vals=msg_vals, **kwargs)
+        if notify_by_email:
```

Comment:
I think that in this case, you also want to disable the notification by inbox, if the user has selected "Handle in Odoo" in its preferences. In that case, you might even want to just avoid calling `_notify_thread` at all? Not sure if it makes sense then.

Reply:
Make sure that if you remove the call to `_notify_thread` in this case, the longpolling is correctly updated somehow.